css-ui-4/Overview.bs

```diff
@@ -2166,81 +2166,91 @@ Document languages should specify
+            appropriate widget with |authorProps| set to |authorProps|, |expected| set to [=kind of widget/textfield=],
+            and |fallback| set to [=kind of widget/none=].
 
         <li>
-            If |element| can be a ''searchfield'', given |computedAppearance|, then return the appropriate widget with
-            |authorProps| set to |authorProps|, |expected| set to searchfield, and |fallback| set to
-            ''appearance/none''.
+            If |element| can be a searchfield, given |computedAppearance|, then return the
+            appropriate widget with |authorProps| set to |authorProps|, |expected| set to
+            [=kind of widget/searchfield=], and |fallback| set to [=kind of widget/none=].
 
         <li>
-            If |element| can be a ''slider-horizontal'', given |computedAppearance|, then return slider-horizontal.
+            If |element| can be a slider-horizontal, given |computedAppearance|, then return
+            [=kind of widget/slider-horizontal=].
 
         <li>
-            If |element| can be a ''menulist-button'', given |computedAppearance|, then return menulist-button.
+            If |element| can be a menulist-button, given |computedAppearance|, then return
+            [=kind of widget/menulist-button=].
 
         <li>
-            If |element| can be a ''menulist'', given |computedAppearance|, then return menulist.
+            If |element| can be a menulist, given |computedAppearance|, then return
+            [=kind of widget/menulist=].
 
         <li>
-            If |element| can be a ''textarea'', given |computedAppearance|, then return the appropriate widget with
-            |authorProps| set to |authorProps|, |expected| set to textarea, and |fallback| set to
-            ''appearance/none''.
+            If |element| can be a textarea, given |computedAppearance|, then return the
+            appropriate widget with |authorProps| set to |authorProps|, |expected| set to [=kind of widget/textarea=],
+            and |fallback| set to [=kind of widget/none=].
 
         <li>
-            If |element| can be a ''progress-bar'', given |computedAppearance|, then return progress-bar.
+            If |element| can be a progress-bar, given |computedAppearance|, then return
+            [=kind of widget/progress-bar=].
 
         <li>
-            If |element| can be a ''meter'', given |computedAppearance|, then return meter.
+            If |element| can be a meter, given |computedAppearance|, then return
+            [=kind of widget/meter=].
 
         <li>
-            Return ''appearance/none''.
+            Return [=kind of widget/none=].
     </ol>
 
-    The appropriate <a>widget</a>, given a map of properties |authorProps|, and an expected
-    kind of widget |expected|, and a fallback kind of widget |fallback|, is as follows:
+    The appropriate <a>widget</a>, given a [=map=] of properties |authorProps|, and an expected
```

Comment:
"appropriate widget" should be its own `<div algorithm="appropriate widget">`, and linked to from the algorithm above